src/encoded/static/components/collection.js

```diff
@@ -76,7 +87,17 @@ class FacetChart extends React.Component {
                             const text = [];
                             text.push('<ul>');
                             for (let i = 0; i < chartData.length; i += 1) {
-                                const searchUri = `${baseSearchUri}&${facet.field}=${encodeURIComponent(chartLabels[i]).replace(/%20/g, '+')}`;
+                                let queryString = '';
+                                if (this.otherTerms.length && this.otherTermsIndex === i) {
+                                    // Handle a click in the "other" term. We need to generate a
+                                    // query string that combines all the "other" terms.
+                                    queryString = generateMultipleQuery(facet.field, this.otherTerms);
```

Comment:
Is this dependent on the loop?  It looks like 
```
queryString = generateMultipleQuery(facet.field, this.otherTerms);
```
could be called once when the chart is created since otherTerms is set in render.